lib/views/help.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var blessed = require("blessed");
+
+var pkg = require("../../package.json");
+
+var HelpView = function HelpView(options) {
```

Comment:
it's not tied to any layouts and doesn't do any resizing - so most of what `BaseView` provides is n/a